src/main/java/org/dependencytrack/model/Policy.java

```diff
@@ -142,6 +142,10 @@ public enum ViolationState {
     @Column(name = "INCLUDE_CHILDREN", allowsNull = "true") // New column, must allow nulls on existing data bases)
     private boolean includeChildren;
 
+    @Persistent
+    @Column(name = "ONLY_FOR_LATEST_PROJECT_VERSION", allowsNull = "true") // New column, must allow nulls for existing data bases
```

Comment:
Consider also using `defaultValue = "false"` here, same as for `Project#isLatest`. Also, can we remove the `_FOR_` part? Seems like an unnecessary filler word for a database column.

Reply:
done